lua/rocks/operations/helpers.lua

```diff
@@ -61,6 +61,10 @@ helpers.install = function(rock_spec, progress_handle)
             table.insert(install_cmd, version)
         end
     end
+    ---@param install_arg string
+    vim.iter(rock_spec.install_args or {}):each(function(install_arg)
```

Comment:
how does this work with rocks-lazy args for instance ? if I add `opt=true` is it still passed on ? in case we worry about conflicts, maybe we can add a delimiter like in the shell: everything after `--` is forwarded to luarocks

Reply:
Yep, `opt=true` doesn't have anything to do with installation, so there's nothing to worry about here.